examples/create_averaged_submissions_from_local_disk.py

```diff
@@ -31,14 +31,14 @@ def run(
 
     evaluation_file_paths_: List[T_path] = list(evaluation_file_paths)
     packages_file_paths_: List[T_path] = list(packages_file_paths)
-    wandb_log_dirs_: List[T_path] = list(wandb_log_dirs)
+    wandb_log_dirs_: List[T_path] = list(wandb_config_file_paths)
 
     submission = AveragedSubmission.from_local_disk(
         submission_name=submission_name,
         task=task,
         evaluation_file_paths=evaluation_file_paths_,
         packages_file_paths=packages_file_paths_,
-        wandb_log_dirs=wandb_log_dirs_,
+        wandb_config_file_paths=wandb_log_dirs_,
```

Comment:
I don't like the naming here as we pass a directories where file paths are expected

Reply:
This function handles both directories and files